docs/src/modules/components/DiamondSponsors.js

```diff
@@ -128,20 +123,15 @@ export default function DiamondSponsors() {
           border: '1px dashed',
           transition: theme.transitions.create(['color', 'border-color', 'background-color']),
           backgroundColor: alpha(theme.palette.primary[50], 0.5),
-          borderColor: (theme.vars || theme).palette.grey[200],
-          boxShadow: `inset 0 1px 1px ${
-            (theme.vars || theme).palette.grey[50]
-          }, 0 1px 2px ${alpha(theme.palette.grey[100], 0.5)}`,
+          borderColor: (theme.vars || theme).palette.divider,
+          boxShadow: `${alpha(theme.palette.primary[50], 0.8)} 0 2px 0 inset, ${alpha(theme.palette.primary[100], 0.4)} 0 -1px 0 inset, ${alpha(theme.palette.primary[100], 0.5)} 0 1px 2px 0`,
```

Comment:
I'd remove the box-shadow here, if it should feel like an empty space, it makes more sense not having it (a little bump on the border color might be needed to): 

<img width="250" alt="Screenshot 2024-03-27 at 17 33 28" src="https://github.com/mui/material-ui/assets/37222944/30051340-6409-4386-871e-4e26e0cba22c">


besides, it looks kinda muddy in the light theme:

<img width="250" alt="Screenshot 2024-03-27 at 17 33 53" src="https://github.com/mui/material-ui/assets/37222944/503458f5-43ee-4daa-88ec-3fbf2817aa7b">

Reply:
Good call, yeah!